### PR TITLE
Fix accessibility and structural issues

### PR DESCRIPTION
**Pull request by m-green**
### Context
We're [auditing and fix accessibility issues in docs that use the Tech Docs Template](https://github.com/alphagov/design-system-team-internal/issues/312).

### Changes proposed in this pull request
- Ensure links are accessible
- Fix some GOV.UK style issues
- Fix typos
- Update GitHub and Heroku instructions to match current sites
- Restructure repo to fix 404s in navigation links

### CODE REVIEW

File: source/task5/task5.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+git clone https://github.com/username/repo.git
+Username: your_username
+Password: your_token
+```
+
+## Deploy with Heroku
+
+1. Go to [www.heroku.com](Heroku) and log in if you're not already logged in.
+
+2. Select **New**, then **Create new app**.
```
Comment (36degrees) [inline source/task5/task5.html.md.erb:143]: We seem to mix using `__` and `**` for emphasis in this file – is that worth fixing whilst we're tidying things up?

File: source/cheatsheets/cheatsheets.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+git branch
+```
+
+Create a new branch:
+
+```shell
+git branch BRANCH-NAME
+```
+
+Switche to the specified branch and update working directory:
```
[the remark this answers is not in the document]
Comment (m-green) [reply source/cheatsheets/cheatsheets.html.md.erb:188]: Oops! Thanks for spotting!

File: source/task5/task5.html.md.erb (hunk 1 above)
Comment (m-green) [reply source/task5/task5.html.md.erb:143]: Good plan, thanks!

File: config/tech-docs.yml [hunk 3]
```diff
@@ -20,8 +20,6 @@ max_toc_heading_level: 2
 # Tracking ID from Google Analytics (e.g. UA-XXXX-Y)
 # ga_tracking_id: UA-XXXX-Y
 
-google_site_verification: "IJ5HOXpZrISM6la-YO_iX0rBUC5YFftpexygcKLsNs4"
```
[the remark this answers is not in the document]
Comment (m-green) [reply config/tech-docs.yml:23]: No problem - will do.

**Comment by m-green** [comment]
> At the minute lots of the URLs have changed and the URLs include duplication, e.g. `/task1.html` has 'moved' to `/task1/task1.html`
> 
> I think all of the following should be moved up a level and their parent directories delete:
> 
> ```
> cheatsheets/cheatsheets.html.md.erb => /cheatsheets.html.md.erb
> nextsteps/nextsteps.html.md.erb     => /nextsteps.html.md.erb
> resources/resources.html.md.erb     => /resources.html.md.erb
> task1/task1.html.md.erb             => /task1.html.md.erb
> task2/task2.html.md.erb             => /task2.html.md.erb
> task3/task3.html.md.erb             => /task3.html.md.erb
> task4/task4.html.md.erb             => /task4.html.md.erb
> task5/task5.html.md.erb             => /task5.html.md.erb
> ```

Good point on this. For some reason I thought the separate folders were a requirement. Will fix!

**Comment by m-green** [comment]
Hi @36degrees - thanks for your feedback on this! I've reorganised and the commits should be better. Let me know if it all looks ok.

File: source/resources.html.md.erb [hunk 4]
```diff
@@ -0,0 +1,16 @@
+---
```
Comment (36degrees) [inline source/resources.html.md.erb:1]: This file has also changed (from `/related-resources.html`) so can we add a redirect for that as well?

Comment (m-green) [reply source/resources.html.md.erb:1]: Ah good point! Fixed in baee611

**Pull request merged**